src/google/google-extractor/api/courses.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+from .api_caller import call_api
+
+
+def request_courses(resource):
+    return call_api(
+        resource.courses().list,
+        {"courseStates": ["ACTIVE"]},
+        "courses",
+    )
+
+
+def courses_dataframe(resource):
```

Comment:
It is trivial enough to read the code and see the difference between these. If using auto-complete, it would be harder to tell what needs to occur. Aside from the question of docstring - which is nice to have but I don't see it as mandatory - I wonder about changing the function name to include a verb, and to help the caller know that this is actually making an API request?

Reply:
changing this naming pattern to "request_xyz_as_df"